task_DEV-3/Program.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;

```

Comment:
Not really. Its here because I remodeled all the branches so that the solution had all the projects, but at the moment the logic of the task_Dev-3 has not been changed, so the merge was not done from the task_ of the DEV-3 in the master. Now I'm working on it. Until I make a merge from the task_Dev-3, this commit will hang. U will not see such things in the future, I promise.